lambdas/service/.chalice/config.json.template.py

```diff
@@ -29,7 +34,7 @@
                 config.cart_export_dss_push_lambda_basename: {
```

Comment:
Use same pattern here, too. All of the `config.…_lambda_basename` properties should disappear.

Reply:
Let's discuss this during after standup because removing these completely for the cart endpoints turns out to be a lot more complicated than it at first appears.